To-List-Project-HTML2.html

```diff
@@ -4,229 +4,30 @@
 <head>
 	<title>Todo List Project</title>
 	<meta charset="utf-8">
-	
-	<style>
-	body {
-	background-color: #FAECA9
-	}
-	
-	.topBar {
-		font-family: sans-serif;
-		font-size: 1em;
-		text-align: center;
-		color: white;
-		position: fixed;
-		top:-3%;
-		left:0%;
-		z-parentListIndex: 100;
-		width: 100%;
-		height: 10%;
-		background-color: #8E2EC2;
-	}
-	.lists {
-		font-family: sans-serif;
-		font-size: 1.5em;
-		top: 5%;
-		margin-left: 10%;
-		margin-right: 20%;
-		margin-top: 5%;
-	
-	}
-	.form {
-		color: #FF8000;
-		font-size: 0.5em;
-	}
-	.subForm {
-		color: #FF8000;
-		font-size: 0.5em;
-	}
-	.accessories {
-		text-align: center;
-	}
-	
-	</style>
-	
+	<link href="todo.css" rel="stylesheet">
+
 </head>
 
 
-<body onload = "load()">
+<body>
 	<div class= topBar>
```

Comment:
the quotes is I believe required, but some browsers will put them in for you. But things might not work for browsers that do not have quotes around class names.

For using hyphens to delimit words, I think it just is the industry standard for css class names